lib/clients/AzureClient.py

```diff
@@ -131,6 +131,7 @@ def get_attached_volumes_for_instance(self, instance_id):
             instance = self.compute_client.virtual_machines.get(
                 self.resource_group, instance_id)
             self.instance_location = instance.location
+            self.availability_zones = instance.zones
```

Comment:
I think it would be better if we populate variable 'self.availability_zones' inside init method. So that it would be inline with all other clients e.g. see [here](https://github.com/cloudfoundry-incubator/service-fabrik-backup-restore/blob/master/lib/clients/AwsClient.py#L36). So, if other method needs the 'zone' it need not to depend on calling get_attached_volumes_for_instance or fetch it again. In top it can be fetched one

Reply:
Agree .

@pritishmishra : can you incorporate this comment ? And is this PR mergeable now i.e can it be merged without the multi-az support not present ?